docs/containers/app-service.md

```diff
@@ -88,6 +89,24 @@ In the previous section, the image is pushed to a remote container registry. Now
 
     ![Web Application](images/app-service/webapp-homepage.png)
 
+## Deploy to ACI
```

Comment:
How about starting with this rather than the current first sentence in the section?
In the previous section, the image was deployed to Azure App Service. Another option is to deploy the image to [Azure Container Instances (ACI)](https://azure.microsoft.com/services/container-instances/).